From 6ee94659087b4e9017166e6f837b49aae6a00ad6 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Mon, 17 Sep 2018 16:49:35 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399-sapphire-excavator-linux: fixes
 the incorrect pdn pin for rk3399 camera

This patch fixes that ethernet is abnormal without connecting the camera,
since the gpio3_a4 is reused by the gmac.

Change-Id: I39cd40f62d706491dac57eda2c43408c4a88e7a0
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 .../arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts
index 10a7f4f7a96e..6b5e3219407f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts
@@ -335,7 +335,7 @@
 				"rockchip,camera_sleep";
 		pinctrl-0 = <&cam0_default_pins>;
 		pinctrl-1 = <&cam0_sleep_pins>;
-		rockchip,pd-gpio = <&gpio3 4 GPIO_ACTIVE_LOW>;
+		rockchip,pd-gpio = <&gpio1 4 GPIO_ACTIVE_LOW>;
 		//rockchip,pwr-gpio = <&gpio3 13 0>;
 		/* mipi switch control*/
 		rockchip,rst-gpio = <&gpio2 10 GPIO_ACTIVE_LOW>;
-- 
2.35.3

